frontend/src/components/pages/ViewDonations.tsx

```diff
@@ -0,0 +1,92 @@
+import {
+  CalendarIcon,
+  ChevronLeftIcon,
+  ChevronRightIcon,
+} from "@chakra-ui/icons";
+import {
+  Box,
+  Button,
+  Flex,
+  HStack,
+  IconButton,
+  Spacer,
+  Stack,
+  Text,
+  VStack
+} from "@chakra-ui/react";
+import React, { useState } from "react";
+import DatePicker, { DateObject } from "react-multi-date-picker";
+import Icon from "react-multi-date-picker/components/icon";
+
+import AdminCalendar from "./AdminCalendar";
+
+const ViewDonations = (): React.ReactElement => {
+  const [selectedDay, setSelectedDay] = useState<
+    Date | DateObject | DateObject[] | null
+  >(new Date());
+  const [test, setTest] = useState<any>(0);
+
+  const changeWeek = (days: number) => {
+    setTest(test + 1); // need this for some reason

```

Comment:
need this for a forced state update since updating `DateObject` from the react-multi-picker-date library does not seem to trigger one